setup.py

```diff
@@ -71,14 +72,27 @@ def build_extension(self, ext):
 except IndexError:
     raise RuntimeError('Unable to determine version.')
 
+install_requires = [
+    'attrs>=17.3.0',
+    'chardet>=2.0,<4.0',
+    'multidict>=4.0,<5.0',
+    'async_timeout>=3.0,<4.0',
+    'yarl>=1.0,<2.0',
+]
+extras_require = {}
+
+setuptools_version_tuple = setuptools.__version__.split('.')
+setuptools_version = (
+    int(setuptools_version_tuple[0]),
+    int(setuptools_version_tuple[1]),
+    int(setuptools_version_tuple[2]),
+)
 
-install_requires = ['attrs>=17.3.0', 'chardet>=2.0,<4.0',
-                    'multidict>=4.0,<5.0',
-                    'async_timeout>=3.0,<4.0',
-                    'yarl>=1.0,<2.0']
-
-if sys.version_info < (3, 7):
-    install_requires.append('idna-ssl>=1.0')
+if setuptools_version >= (36, 2, 2):
```

Comment:
environment markers are supported since 20.5 version, but really get worked since 21.0.  No need to require such high setuptools version.

Reply:
Also, version 21.0 released two years ago. Once again, do we need any checks against setuptools version at all? Unlikely any sane user will install latest aiohttp having legacy package management toolset.